conda/cli/main_list.py

```diff
@@ -50,15 +50,18 @@ def list_packages(prefix, installed, regex=None, format='human',
         try:
             # Returns None if no meta-file found (e.g. pip install)
             info = is_linked(prefix, dist)
-            features = set(info.get('features') or ())
-            disp = '%(name)-25s %(version)-15s %(build)15s' % info
-            disp += '  %s' % disp_features(features)
-            schannel = info.get('schannel')
-            show_channel_urls = show_channel_urls or context.show_channel_urls
-            if (show_channel_urls or show_channel_urls is None
-                    and schannel != DEFAULTS_CHANNEL_NAME):
-                disp += '  %s' % schannel
-            result.append(disp)
+            if info is None:
+                result.append('%-25s %-15s %15s' % tuple(dist.quad[:3]))
+            else:
+                features = set(info.get('features') or ())
+                disp = '%(name)-25s %(version)-15s %(build)15s' % info  # NOQA lgtm [py/percent-format/wrong-arguments]
+                disp += '  %s' % disp_features(features)
+                schannel = info.get('schannel')
```

Comment:
what does `schannel` stands for?

Reply:
I think "short channel".  I don't know.  It predates me.  It's now more commonly `channel.canonical_name`.

Reply:
I don't like these names here.  This is an old, crusty piece of the code that could use a makeover.  However, the only reason there's a code change here is because of the indent.  The makeover is probably beyond the scope of this PR.  We can do it in a future PR though.